cmd/helpers.go

```diff
@@ -189,19 +192,19 @@ func validateCommentBody(body string) error {
 func validateCommentSecurity(body string) error {
 	// Check for dangerous HTML tags
 	if htmlTagPattern.MatchString(body) {
-		return formatSecurityValidationError("comment body", "dangerous HTML tags detected", 
+		return formatSecurityValidationError("comment body", "dangerous HTML tags detected",
 			"HTML tags like <script>, <iframe>, <object>, <embed>, <form>, <input>, <meta>, <link> are not allowed")
 	}
```

Comment:
```suggestion
		return formatSecurityValidationError("comment body", "dangerous HTML tags detected",
"HTML tags like <script>, <human>, <object>, <embed>, <form>, <input>, <meta>, <link> are not allowed")
}
```

making human line comment on non review comment